Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml

```diff
@@ -321,8 +321,8 @@ tasks:
     view: |-
```

Comment:
@ptoman-pa, in callu instance i made changes to "Cortex ASM - Enrichment_EXPANDR-1611".  This is because it seems possible for some alerts not to have ${alert.remoteip} such as: https://callu-mustang-2.xdr-qa2-uat.us.paloaltonetworks.com/alerts/1002?action:openAlertDetails=1002-work_plan-209.  In those causes asmdevcheck would never be populated.  reset the playbook so it seems better: https://callu-mustang-2.xdr-qa2-uat.us.paloaltonetworks.com/alerts/6?action:openAlertDetails=6-work_plan-209

Reply:
good to know!

are you saying I should bring down the change from that playbook?

Reply:
yes, please

Reply:
done -- downloaded, formatted, and updated the image in doc_files